roles/rsyslog/roles/input_roles/journald/defaults/main.yaml

```diff
@@ -0,0 +1,367 @@
+---
+# journald configuration
+# ---------------------
+
+# journald rpm packages
+# adding rsyslog_logging_packages
+# ------------------
+
+# .. envvar:: rsyslog_logging_packages
+#
+# List of rpm packages for Common Logging.
+rsyslog_journald_packages: ['rsyslog-mmnormalize', 'rsyslog-mmjsonparse', 'libfastjson', 'liblognorm', 'libestr']
+rsyslog_journald_prereq_packages: []
+
+# Available configuration set
+# ----------------------------
+
+rsyslog_unprivileged: 'False'
+
+# journald Rsyslog configuration rules
+# ---------------------------
+
+rsyslog_journald_rules:
+  - '{{ rsyslog_conf_journald_main_modules }}'
+  - '{{ rsyslog_conf_journald_formatting }}'
+  - '{{ rsyslog_rulebase_viaq_parse_json }}'
+  - '{{ rsyslog_json_viaq_normalize_level }}'
+  - '{{ rsyslog_json_viaq_prio_to_level }}'
+
+# Default journald configuration options
+# -----------------------------------
+
+rsyslog_conf_journald_main_modules:
+
+  - name: 'journald_main'
+    type: 'modules'
+    sections:
+
+      - options: |-
+          # Read from journal
+          module(load="imjournal" StateFile="{{ rsyslog_work_dir }}/imjournal.state" RateLimit.Burst="{{ imjournal_ratelimit_burst|default(1000000) }}" RateLimit.Interval="{{ imjournal_ratelimit_interval|default(10) }}" PersistStateInterval="1000" )
+
+      - options: |-
+          # Normalize logs
+          module(load="mmnormalize")
```

Comment:
This and the following aren't really generic journald formatting/normalizing - this is very specific to the viaq common logging/openshift use case (which is the ovirt journald -> common logging use case as well).  It is perfectly acceptable to pass the journald logs as-is to the destination, with no further processing.  The `prio_to_level` lookup table is useful to have here, but again, not everyone will want to use it - they may want to pass the raw `PRIORITY` numerical value to the ultimate log destination.